apps/condo/domains/acquiring/schema/SumPaymentsService.test.js

```diff
@@ -0,0 +1,140 @@
+/**
+ * Generated by `createservice acquiring.SumPaymentsService`
+ */
+
+const { makeClient } = require('@condo/keystone/test.utils')
+
+const { makePayer, createTestPayment } = require('../utils/testSchema')
+const { SUM_PAYMENTS_MUTATION } = require('../gql')
+const Big = require('big.js')
+const { makeClientWithSupportUser, makeClientWithNewRegisteredAndLoggedInUser } = require('../../user/utils/testSchema')
+const { createTestOrganizationEmployeeRole, createTestOrganizationEmployee } = require('../../organization/utils/testSchema')
+ 
+describe('SumPaymentsService', () => {
+    test('admin: sum one payment', async () => {
```

Comment:
Tests needs to be split to groups:

1. Access Group, where you check whether this query can be invoked by Admin, Support, Empolyee, or else. You also do all access-related stuff here.
2. Logic Group, where you check if business logic is correct. 

Please check out other custom queries and mutations